### PR TITLE
Update copy and layout for the Learn More screen

### LINKED ISSUE #219: Update copy and design for Learn More screen
### Reason

New user requirements for the rewards pilot need to be detailed

### Description

The Learn More (offer detail) screen needs to detail the new requirements for users to receive payouts, including listing out all the supported retailers / merchants.

This is where we need to expound on anything that can't reasonably fit on the other screens.

### Success Criteria

- [x] Design new layout based on wireframes (see references)
- [x] Compile full list of supported merchants for account linking
- [x] Compile full list of supported merchants for email linking
- [x] Acquire icons to use for retailers
- [x] Write new copy
- [x] New screen / page fully implemented in example app

### Additional Information

- https://github.com/tiki/apps-receipt-capacitor/issues/186#issuecomment-1728963476
- Supported for account linking: https://ereceipts.blinkreceipt.com/account-linking
- Supported for email linking: https://ereceipts.blinkreceipt.com/us
- Markdown source: https://github.com/tiki/apps-receipt-capacitor/blob/main/src/assets/learn-more.md
- Vue source: https://github.com/tiki/apps-receipt-capacitor/blob/main/src/components/learn-sheet.vue

### PR DESCRIPTION
**Pull request by timoguin**
Closes #219.

## Changed

- Updates copy and layout for the Learn More screen
- Enables support for HTML (and other options) in the Markdown parser for the Learn More sheet

---

# Screen Recording

https://github.com/tiki/apps-receipt-capacitor/assets/671968/6d798b4c-a902-4c06-83aa-9321e65f74fd

### CODE REVIEW

File: Makefile
Comment (mike-audi) [inline Makefile:1]: This doesn't belong in this PR. Please remove

File: src/components/sheet/sheet-learn.vue [hunk 1]
```diff
@@ -4,16 +4,24 @@
...
-defineEmits(["back"]);
+defineEmits(["back", "close"]);
 defineProps({
   markdown: {
     type: String,
     required: true,
   },
 });
+
+const options: Options = {
```
Comment (mike-audi) [inline src/components/sheet/sheet-learn.vue:18]: this should be in-line

Comment (timoguin) [reply src/components/sheet/sheet-learn.vue:18]: Give me an example of what you mean.

**Comment by mike-audi** [comment]
putting this here @timoguin since it's hard to find in the requested changes.

Unless I'm missing it, you're missing a couple key sections:

how do I opt out/change my mind etc.
how to withdraw funds
how is my data used
there may be other key terms of the program that need explaining from the TOS that aren't coming to mind

**Comment by mike-audi** [comment]
I'm going to merge this in as is and open a new story for adding additional details to page.

**Comment by timoguin** [comment]
> I'm going to merge this in as is and open a new story for adding additional details to page.

Thanks. I was actively writing out a response to suggest the same.
